src/Driver/AbstractMySQLDriver.php

```diff
@@ -95,7 +95,7 @@ private function getOracleMysqlVersionNumber(string $versionString): string
 
     /**
      * Detect MariaDB server version, including hack for some mariadb distributions
-     * that starts with the prefix '5.5.5-'
+     * that starts with the prefix '5.5.5-' (fixed since PHP 8.0.16/8.1.3)
```

Comment:
This comment doesn't help understand the code since there's no logic specific to the PHP version.